tests/utils/profilehandler/profile_handler.go

```diff
@@ -446,6 +474,33 @@ func GenerateClusterCreateFlags(profile *Profile, client *rosacli.Client) ([]str
 			)
 
 		}
+		if profile.ClusterConfig.SharedVPC {
```

Comment:
ClusterConfig is not set. You can not verify the shared vpc cluster in day1-post case